WebUI/Pages/GameDetails.razor

```diff
@@ -89,9 +124,19 @@ else
         StateHasChanged();
     }
 
+    private async void SubmitReview()
+    {
+        await reviewService.AddReview(review);
+    }
+
```

Comment:
You're not using this method anywhere. You should add `@onclick` to the button with the call to this method. Also, you're never assigning anything to review after its declaration. Make sure to grab the values from the inputs and create a new Review using them.